docassemble/ALToolbox/ThreePartsDate.py

```diff
@@ -180,18 +223,21 @@ def validate(cls, item):
             try:
                 date = as_datetime(item)
             except Exception as error:
-                msg = f"{ item } {word('is not a valid date')}"
-                raise DAValidationError(msg)
+                ex_msg = f"{ item } {word('is not a valid date')}"
+                raise DAValidationError(ex_msg)
             if matches_date_pattern:
                 date_diff = date_difference(starting=date, ending=today())
                 if date_diff.days >= 0.0:
                     return True
                 else:
                     raise DAValidationError(
-                        word("Answer with a <strong>date of birth</strong>")
+                        word(
+                            f"Answer with a <strong>date of birth</strong> ({date} is in the future)"
+                        )
                     )
             else:
-                msg = (
-                    f"{ item } {word('is not a valid <strong>date of birth</strong>')}"
+                msg = check_empty_parts(
+                    item, default_msg="is not a valid <strong>date of birth</strong>"
```

Comment:
Again this might not be a translatable fragment. Just worth noting before we start adding any entries in word.yml

Reply:
This on is a bit more complicated though, as it's directly concatenated with the invalid date. We can't know exactly where the date is going to be.

Is it normal for the result of the `word` function to still have formatting variables, like `"The number is  is {my_var}"`? That would be doable I guess.